.github/stale.yml

```diff
@@ -0,0 +1,65 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 700
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 60
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - architectures-enhancements
+  - enhancement
```

Comment:
What about adding a new label "never-stale" for those that we think are relevant in foreseeable future?

Reply:
Also `New Architecture` and `New File-Format`, probably also `help-wanted`.

Reply:
> What about adding a new label "never-stale" for those that we think are relevant in foreseeable future?

If that's the case, you can just update the issue as soon as you receive the notification. Otherwise it's too easy to just set the label on every issue and they will always remain there.